web_server/friendship/models.py

```diff
@@ -1,3 +1,23 @@
 from django.db import models
+from uuid import uuid4
+from users.models import Author
+
 
 # Create your models here.
+
+
+class Friend(models.Model):
+
+    author_id = models.URLField(max_length=500)
+    friend_id = models.URLField(max_length=500)
+
+    class Meta:
+        unique_together = (("author_id", "friend_id"),)
+
+
+class FriendRequest(models.Model):
```

Comment:
FriendRequest wasn't one of the models we discussed during our meeting last week. I'm fine if we add this model, but I'm uncertain how you plan to use it, and your commit lacks documentation regarding it. Please document this model.

Reply:
Friend request is the model that is intended to store incoming friend requests. 
`CREATE TABLE IF NOT EXISTS "friendship_friendrequest" ("id" integer NOT NULL PRIMARY KEY AUTOINCREMENT, "from_id" varchar(500) NOT NULL, "to_id" varchar(500) NOT NULL);
CREATE UNIQUE INDEX "friendship_friendrequest_from_id_to_id_c10e9741_uniq" ON "friendship_friendrequest" ("from_id", "to_id");`

Reply:
so the friend request is sent from `from_id` to  `to_id`. Both are URLfield.